### PR TITLE
BE Scores Aggregation

### PR DESCRIPTION
**Pull request by maamalama**
BE scores aggregation

### CODE REVIEW

File: valhalla/jawn/src/controllers/public/experimentController.ts [hunk 1]
```diff
@@ -80,6 +89,7 @@ export class ExperimentController extends Controller {
       requestBody.filter,
       requestBody.include ?? {}
     );
+    console.log(JSON.stringify(result));
```
Comment (chitalian) [inline valhalla/jawn/src/controllers/public/experimentController.ts:92]:
```suggestion
```

File: valhalla/jawn/src/lib/stores/experimentStore.ts [hunk 2]
```diff
@@ -215,6 +260,12 @@ async function enrichExperiment(
     }
   }
 
+  const experimentScores = getExperimentScores(experiment);
+
+  if (!experimentScores.error && experimentScores.data) {
```
Comment (chitalian) [inline valhalla/jawn/src/lib/stores/experimentStore.ts:265]: This is kind of a nit, but i think this something as you progress your career is to feel very allergic to mutations. These things can cause many bugs. as more code get's inected. 

I would suggest a different pattern...

```ts
if (!experimentScores.error && experimentScores.data) {
   return {
      scores: experimentScores.data,
      ...experiment
   }
} else {
   return experiment;
}
```


This is not a big deal, but just something I wanna bring up now for future reference

File: valhalla/jawn/src/lib/stores/experimentStore.ts [hunk 3]
```diff
@@ -328,3 +379,111 @@ export const ServerExperimentStore: {
...
+
+  return ok(scores);
+}
+
+function modelCost(modelRow: {
+  model: string;
+  provider: string;
+  sum_prompt_tokens: number;
+  sum_completion_tokens: number;
+  sum_tokens: number;
```
Comment (chitalian) [inline valhalla/jawn/src/lib/stores/experimentStore.ts:476]: it doesn't look like sumTokens is used

File: valhalla/jawn/src/lib/stores/experimentStore.ts [hunk 4]
```diff
@@ -328,3 +379,111 @@ export const ServerExperimentStore: {
     );
   },
 };
+
+function getExperimentScores(
```
Comment (chitalian) [inline valhalla/jawn/src/lib/stores/experimentStore.ts:383]: Can you please make this more functional, and less scripty

File: valhalla/jawn/src/lib/stores/experimentStore.ts (hunk 2 above)
Comment (maamalama) [reply valhalla/jawn/src/lib/stores/experimentStore.ts:265]: fixed

File: valhalla/jawn/src/lib/stores/experimentStore.ts [hunk 5]
```diff
@@ -328,3 +382,123 @@ export const ServerExperimentStore: {
...
```
Comment (chitalian) [inline valhalla/jawn/src/lib/stores/experimentStore.ts:386]: excellent job!!

**Review by chitalian — APPROVED** [review]

**Pull request merged**